Header header

# GPGGA messages
float64 UTC_Time
float64 Latitude
bool NS_ind
float64 Longitude
bool EW_ind
int32 Pos_Fix_ind
int32 Num_of_Satellite
float64 HDOP
float64 Altitude



# GPHDT messages
float64 Heading
bool TF_ind